urlwatch

```diff
@@ -236,12 +242,45 @@ if __name__ == '__main__':
             cache_dir = options.cache
             log.info('using {} as cache directory'.format(cache_dir))
                 
+    if options.pushover_file:
+	options.pushover = True
             
     # Created all needed folders
     for needed_dir in (urlwatch_dir, cache_dir, scripts_dir):
         if not os.path.isdir(needed_dir):
             os.makedirs(needed_dir)
 
+    if options.pushover:
+        import ConfigParser
```

Comment:
Indentation (use 4 spaces everywhere). And don't do imports inside the code, move the imports to the top of the file (in case of chump, you can set `chump = None` in the `except ImportError:` block and then check it here `if chump is None: [...]`.